custom-elements/reactions/HTMLTrackElement.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<title>Custom Elements: CEReactions on HTMLTrackElement interface</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<meta name="assert" content="kind, src, srclang, label, default of
+  HTMLTrackElement interface must have CEReactions">
+<meta name="help" content="https://html.spec.whatwg.org/#the-track-element">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/custom-elements-helpers.js"></script>
+<script src="./resources/reactions.js"></script>
+
+<video id="video" autoplay controls
+  onplaying="this.onplaying = null; this.pause(); this.currentTime = 0;">
+  <source src="/media/white.webm" type="video/webm">
+  <source src="/media/white.mp4" type="video/mp4">
+</video>
+
+<script>
+
+function getParentElement() {
+  return document.getElementById('video');
+}
+
+function setAttributes(instance, attributes) {
+  for (let [attribute, value] of attributes) {
+    instance.setAttribute(attribute, value);
+  }
+}
+
+testReflectAttributeWithDependentAttributes(
```

Comment:
I have a probably-dumb question. Why do all these tests talk about "dependent attributes"? The CEReactions should fire if you only change metadata, without also changing src. Why do all these tests also change a bunch of other attributes?